symfony/messenger/4.1/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "#MESSENGER_ADAPTER_DSN": "amqp://guest:guest@localhost:5672/%2f/messages"
```

Comment:
The `#` looks weird here. Keep in mind that this is going to be used for more than the `.env` file.

I would do something like: `"#1": "MESSENGER_ADAPTER_DSN amqp://guest:guest@localhost:5672/%2f/messages",` instead.

Reply:
Well, [that's what we do in the `framework-bundle`](https://github.com/symfony/recipes/blob/bd247214b647c7974bb2e21ac1be8854fdbfdc47/symfony/framework-bundle/3.3/manifest.json#L17) as well.

Reply:
I'm not sure what you mean by `#1` as a key 🤔

Reply:
We use the same here: https://github.com/symfony/recipes/blob/bd247214b647c7974bb2e21ac1be8854fdbfdc47/symfony/framework-bundle/3.3/manifest.json#L17

should we change it? Not sure to understand what you mean by
> Keep in mind that this is going to be used for more than the .env file.

?

Reply:
We are using the `env` config to update `phpunit.xml` as well. So, using `#` to mean "comment" is wrong. `#1` is a convention to tell the parser that this is a comment, which is then interpreted depending on where we are using the env var.

Reply:
> `#1` is a convention to tell the parser that this is a comment, which is then interpreted depending on where we are using the env var.

Alright, I'm going to change it that way; though... is it documented somewhere? 🤔